templates/vue-starter-template/app/composables/useCmsHead.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  getCategoryImageUrl,
+  getSmallestThumbnailUrl,
+  isLandingPage,
+  isProduct,
+} from "@shopware/helpers";
+import type { Schemas } from "#shopware";
+
+export type UseCmsHeadReturn = ReturnType<typeof useCmsHead>;
+
+type CmsPageEntity =
```

Comment:
why not use Schemas["CmsPage"] ?

Reply:
https://github.com/shopware/frontends/pull/987/files#diff-810b2e943f5c2779277b977880c39cd7d94df321439f7645839a466e7047c8fa

Types do not overlap

Reply:
so what should we do to fix it? What is needed on the API declarations to make it correct?

Reply:
We cannot replace `CmsPageEntity` with the `CmsPage` type because it is not a union for `Category`, 
  `LandingPage` and `Product` types